src/autorest-core/lib/pipeline/artifact-emitter.ts

```diff
@@ -16,18 +17,30 @@ function IsOutputArtifactOrMapRequested(config: ConfigurationView, artifactType:
 
 async function EmitArtifactInternal(config: ConfigurationView, artifactType: string, uri: string, handle: DataHandle): Promise<void> {
   config.Message({ Channel: Channel.Debug, Text: `Emitting '${artifactType}' at ${uri}` });
+  const emitArtifact = (artifact: Artifact): void => {
+    if (artifact.uri.startsWith("stdout://")) {
```

Comment:
So, in AaaL, the consumer sets the `output-folder` to `stdout://` and they get all the files via messages?